sql/moz-fx-data-shared-prod/monitoring/shredder_run_stats/view.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.monitoring.shredder_run_stats`
+AS
+SELECT
+  shredder_run_date,
+  MIN(start_time) AS start_time,
+  MAX(end_time) AS end_time,
+  TIMESTAMP_DIFF(MAX(end_time), MIN(start_time), HOUR) AS run_time_hours,
+  TIMESTAMP_DIFF(MAX(end_time), MIN(start_time), HOUR) / 24 AS run_time_days,
+  SUM(total_slot_ms) / 1000 / TIMESTAMP_DIFF(MAX(end_time), MIN(start_time), SECOND) AS avg_slots,
+  CASE
+    WHEN task_id LIKE 'moz-fx-data-shared-prod.telemetry_stable.main_v5%'
```

Comment:
you may want to leave the actual version out
```suggestion
    WHEN task_id LIKE 'moz-fx-data-shared-prod.telemetry_stable.main_v%'
```